### PR TITLE
Resolve default countryCodeSize when not set on commissioner

### LINKED ISSUE #16723: Country Code default length is not set correctly upon GetCountryCode() Read failure
Problem
Within the kConfigRegulatory commissioning step, an attempt to read the country code from NVS is performed. If this has not be previosuly written, the Linux NVS driver implementation will overwrite the passed in actualCountryCodeSize to 0, which is sent to the DUT.

On the DUT, upon reception of this command we attempt to store the regulatory information:
https://github.com/project-chip/connectedhomeip/blob/621b672f0888e37605df5c2d8f27d31a594e4d79/src/platform/DeviceControlServer.cpp#L57

As we are attempting to store a 0 length attribute, the write end in a failure and the commissioning procedure is halted.

Change overview
Update commissioner to correctly set the default country code size.

Testing
Linux RPI based CHIP-tool - ble-thread pairinig using out of box configuration on CC13x2x7 device.

### PR DESCRIPTION
**Pull request by adabreuti**
#### Problem
* Fixes #16723 

#### Change overview
Update commissioner to correctly set the default country code size.

#### Testing
Linux RPI based CHIP-tool - ble-thread pairinig using out of box configuration on CC13x2x7 device.

### CODE REVIEW

**Review by msandstedt — APPROVED** [review]

File: src/controller/CHIPDeviceController.cpp [hunk 1]
```diff
@@ -1719,6 +1719,7 @@ void DeviceCommissioner::PerformCommissioningStep(DeviceProxy * proxy, Commissio
 #endif
         if (status != CHIP_NO_ERROR)
         {
+            actualCountryCodeSize = 2;
```
Comment (bzbarsky-apple) [inline src/controller/CHIPDeviceController.cpp:1722]: Doesn't this also need to put "XX" back in the buffer?  Nothing guarantees buffer is unmodified by the failing call...

**Comment by adabreuti** [comment]
@bzbarsky-apple @msandstedt I believe the failing PR check is infrastructure related and not related to the changes made in this PR. If you agree, would you be able to merge it?

File: src/controller/CHIPDeviceController.cpp [hunk 2]
```diff
@@ -1719,7 +1719,9 @@ void DeviceCommissioner::PerformCommissioningStep(DeviceProxy * proxy, Commissio
 #endif
         if (status != CHIP_NO_ERROR)
         {
-            ChipLogError(Controller, "Unable to find country code, defaulting to XX");
+            actualCountryCodeSize = 2;
+            memset(countryCodeStr, 0x58, actualCountryCodeSize);
```
Comment (bzbarsky-apple) [inline src/controller/CHIPDeviceController.cpp:1723]:
```suggestion
            memset(countryCodeStr, 'X', actualCountryCodeSize);
```

**Review by bzbarsky-apple — APPROVED** [review]

**Comment by bzbarsky-apple** [comment]
> not related to the changes made in this PR

Agreed.  If it fails again, please ping me on Slack and I will retrigger as needed (and bump job timeouts as needed....)

**Pull request merged**